src/sentry/utils/snuba.py

```diff
@@ -58,6 +58,122 @@ def timer(name, prefix='snuba.client'):
 )
 
 
+def transform_aliases_and_query(**kwargs):
```

Comment:
So we already have this `get_snuba_translators` method which returns a `forward()` and `reverse()` translation function that can be used to do any translations to queries before they go to snuba (and transformations on snuba results before they are returned to the caller. 

I would definitely prefer that we continue to encapsulate all our transformation logic in there and not add other wrapper functions around the query method.

Reply:
Ah, I guess one wrinkle is that the `forward()` translator is only used to translate the filters, and not the other parameters like groupby, selected_columns etc. 

I guess this can go out as is if it's urgent, but I'd still like to eventually put all the translation logic in one place.